From 9dd708ba609b826c07fa91d8a5ee56b368ae8183 Mon Sep 17 00:00:00 2001
From: "madhurki@xilinx.com" <madhurki@xilinx.com>
Date: Mon, 12 Sep 2016 17:14:34 -0700
Subject: [PATCH 6/6] staging: mali: r7p0: page_cache_release() to put_page()

09cbfea replaced page_cache_release() to put_page().

Signed-off-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: madhurki@xilinx.com <madhurki@xilinx.com>
Upstream Status: Pending
---
 driver/src/devicedrv/mali/linux/mali_memory_swap_alloc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/driver/src/devicedrv/mali/linux/mali_memory_swap_alloc.c b/driver/src/devicedrv/mali/linux/mali_memory_swap_alloc.c
index 1aea47b..068e4d0 100755
--- linux/mali_memory_swap_alloc.c
+++ b/linux/mali_memory_swap_alloc.c
@@ -183,7 +183,7 @@ static void mali_mem_swap_out_page_node(mali_page_node *page_node)
 	dma_unmap_page(&mali_platform_device->dev, page_node->swap_it->dma_addr,
 		       _MALI_OSK_MALI_PAGE_SIZE, DMA_TO_DEVICE);
 	set_page_dirty(page_node->swap_it->page);
-	page_cache_release(page_node->swap_it->page);
+	put_page(page_node->swap_it->page);
 }
 
 void mali_mem_swap_unlock_single_mem_backend(mali_mem_backend *mem_bkend)
-- 
2.7.4

